Gemfile

```diff
@@ -20,7 +20,7 @@ gem 'activesupport'
 gem 'actionpack'
 
 gem 'grape-erb'
-gem 'geocoder', github: 'Mapotempo/geocoder'
+gem 'geocoder', git: 'https://github.com/Mapotempo/geocoder.git'
```

Comment:
Why did we need this? Bundler document says that `github: 'Mapotempo/geocoder'` and `git: 'git://github.com/Mapotempo/geocoder.git'` are identical and the first one is a shortcut for the latter one. 

> If you are getting your gems from a public GitHub repository, you can use the shorthand
> 
> `gem 'rack', github: 'rack/rack'`

https://bundler.io/guides/git.html

We are just pulling from public repositories and never writing anyways.

Reply:
It's because build fails now since GitHub have changed, it's the same on every projects. 

 Failing exemple : https://github.com/Mapotempo/geocoder-api/runs/5627507763?check_suite_focus=true

Reply:
Cool, it was not obvious what was the "fix" that's why I asked.